lib/kaname.rb

```diff
@@ -7,3 +9,5 @@
 
 module Kaname
 end
+
+Kaname::Config.setup unless ENV['RACK_ENV'] == 'test'
```

Comment:
minitest will load `minitest_helper.rb` first on running test suite.

https://github.com/yaocloud/kaname/blob/master/test/minitest_helper.rb

`Kaname::Config.setup` can be moved to that location.